parquet-hadoop/src/main/java/org/apache/parquet/hadoop/ParquetFileReader.java

```diff
@@ -1464,7 +1464,7 @@ protected PageHeader readPageHeader(BlockCipher.Decryptor blockDecryptor, byte[]
      */
     private void verifyCrc(int referenceCrc, byte[] bytes, String exceptionMsg) {
       crc.reset();
-      crc.update(bytes);
+      crc.update(bytes, 0, bytes.length);
```

Comment:
Changed to adopt a Java 8 API, to be consistent with the pom

Reply:
This is unrelated, I would prefer to update this in another PR.

Reply:
Agreed, committed to revert this change